src/pages/troubleshooting/troubleshooting-timeouts.md

```diff
@@ -0,0 +1,39 @@
+[Manage Access](/security/manage-access/) page for details.
+## Load balancer timeouts
+Ingress controllers on openshift are tied to load balancers that expose the routes to the outside the cluster network.  These are configured in the ingress controller or in the cloud specific load balancer settings.  On AWS OCP if the ingress controller is of type Internal you can set the timeout on the Classic load balancer associated with the domain name in the AWS Console on its Attributes settings.  In OCP 4.11 and later this can be configured in the ingress controller yaml itself - see [ocp docs](https://docs.openshift.com/container-platform/4.11/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.html) for details. 
+
+## ACD timeouts
+The default timeout between ACD and its microservices is 60 seconds.  If a annotator in the flow takes longer than this ACD will return a 504 Gateway Timeout error back to the caller.  This timeout can be adjusted by adding this environment variable to the merative-acd-acd deployment 
+
+| env variable | description | default (if not set)|
+|----------|-------------|---------------------|
+|clinical_data_annotator_acd_api_request_timeout_read | timeout in milliseconds for any microservice call from acd | 60000
+
+## Reduction of document size
+One way to reduce timeouts on the client side is to break the document into smaller chunks to send to ACD.  This is pretty straightfoward but care should be taken to try to break the document on a page or sentence boundary so the syntatical and lexical parsing is maintained.  Sometimes there are natural breaks in large documents or headers that can be scanned for to assist with this.
+
+## Reducing the wait time in service queues
+If the ACD system is under load it has queues to help buffer requests (too many concurrent requests can cause the processes in ACD to run out of memory and crash so there are safeguards to limit the maximum number of concurrent requets each process can handle at once).  The system doesn't evenly load balance or choose the instance of a service with the least number of active requests so some buffer is put in to avoid errors and this is a request queue.  If any ACD service is experiencing persistent high queue lengths it will slow down processing.  To see if documents are queueing up in ACD or any of its microservices, look at the metric *clinical_data_annotator_api_queued_time_seconds*  (see [Enabling and Configuring ACD prometheus metrics](/troubleshooting/logging-monitoring/#enabling-and-configuring-acd-prometheus-metrics) for metric enablement) or search the logs for the message "Concurrent event". To remedy this add more ACD replicas to your ACD instance (which generally requires more nodes to run on), or reduce the number of documents sent to the ACD instance at once.
```

Comment:
On the phrase "... to avoid errors and this is a request queue", should "and this is a request queue" be dropped or rephrased? Not sure what message was intended with that part of the phrase.